packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,13 @@
+In this tutorial, we will be exploring different ways of rendering a list of
+items in Lit!
+
+Let's begin by looking a how we might render a list of names from an array.
+We've already set up a custom element with a `names` property that is an
+array of strings.
+
+We can utlize the `map` array prototype method to generate a list item
```

Comment:
Fom here i'll just leave comments saying "we" or "lets" so you can keep track on where this stuff is